linux/user-and-file-management/bash-environment/what-are-environment-variables.md

```diff
@@ -15,77 +9,67 @@ type: normal
 
 category: must-know
 
-inAlgoPool: false
-
 tags:
-
   - introduction
-
+  - linux
+  - environment
+  - variables
 
 
 ---
 
-# What Are Environment Variables?
+# Environment Variables
 
 ---
 ## Content
 
-An *environment variable* is a dynamic, named variable that can affect the way processes behave on a computer.  For example, a process might look up the `HOME` environment variable to determine the current user's home directory.
-
-### Common Environment Variables
+An *environment variable* is a variable that affects the way processes behave on a computer. 
 
-On Linux and other Unix-based operating systems, common environment variables include:
-
-- `HOME`, which contains the current user's home directory
-- `PATH`, which contains a list of directory paths.  When a user types a command without providing the full path, `bash` will look at the directories in the `PATH` environment variable to see if they contain the given command.
-- `PS1`, which specifies how the prompt is displayed
-- `EDITOR`, which specifies the default text editor.
-- `LANG`, which specifies the user's language, e.g., a value of `pt_BR` signifies that the user prefers Brazillian Portuguese.
-
-### Displaying Environment Variables
+For example, a process might look up the `HOME` environment variable to determine the current user's home directory.
```

Comment:
This isn't really correct. An "environment" variable describes its availability, not its capability.

See: https://www.digitalocean.com/community/tutorials/how-to-read-and-set-environmental-and-shell-variables-on-a-linux-vps

Wikipedia has a similar definition but they get away with it because they say "can affect" instead of "affects", which is technically correct but not the point.

```suggestion
An *environment variable* is a variable that is available to all the processes running in the shell.

> 💡You can think of environment variables like global variables

Linux contains many default environment variables.

As an example, the `HOME` environment variable holds the current user's home directory.

> 💡Environment variables are usually named with all uppercase letters to differentiate them from regular variables.
```